openreview/venue_request/process/deployProcess.py

```diff
@@ -47,8 +47,8 @@ def process(client, note, invitation):
     readers = [conference.get_program_chairs_id(), SUPPORT_GROUP]
 
     client.post_invitation(openreview.Invitation(
-        id = SUPPORT_GROUP + '/-/Request' + str(forum.number) + '/Revision',
-        super = SUPPORT_GROUP + '/-/Revision',
+        id = SUPPORT_GROUP + '/-/Request' + str(forum.number) + '/Venue_Update',
```

Comment:
Why do you change the name?

Reply:
That is to avoid confusions. We now have another Revision in `Revision Stage` also. I mentioned this in the PR description as well. 
Let me know if you disagree with the change.

Reply:
What if we call the other invitation `Submission Revision Stage`? It more clear for the user that is a stage to enable revision on submissions. @celestemartinez what do you think?

Reply:
I actually named it exactly that earlier. But then I felt that it seems too long in a button. Anyway, if everyone agrees on using this, I can make the changes for that.
Also, the changes would mean that there will be a button called `Revision` and then another called `Submission Revision Stage`. While the latter is really clear in its objective, the former is still confusing as to exactly what it affects. And for that reason, I would like to retain the change of `Revision` to `Venue Update`. Your thoughts @melisabok and @celestemartinez ?

Reply:
I think you should use Submissions Revision Stage because there will be Review Revision Stage, etc. Unless we want all the configuration in a single invitation. Another option is to call it Submission Stage. 

I think keeping the current invitation as Revision is fine.

Reply:
I agree that keeping the venue update invitation as Revision should be fine, but maybe we can mention this button somewhere in the comment to PCs after the venue is deployed:
> If you need to make a change to the information provided in your request form, please feel free to revise it directly 
using the 'Revision' button.

or something like that

Reply:
Thanks, I have made these changes.